first_ruby/public/robots.txt

```diff
@@ -0,0 +1 @@
+# See https://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
```

Comment:
Whats this file for?

Reply:
To give instruction about their site to web robots which called Robots Exclusion Protocol.